File: Extensions/Tessa.Extensions.Client/UI/PnrErrandUIExtension.cs
using System.Threading.Tasks;
using Tessa.Cards;
using Tessa.UI.Cards;

namespace Tessa.Extensions.Client.UI
{
    public sealed class PnrErrandUIExtension : CardUIExtension
    {
        public override Task Initialized(ICardUIExtensionContext context)
        {
            Card card = context.Card;
            ICardModel cardModel = context.Model;

            // Создание карточки
            if (card.StoreMode == CardStoreMode.Insert)
            {
                // Заполнить поле Контролер значением из поля Автор поручения
                cardModel.Card.Sections["PnrErrands"].Fields["ControllerID"] = cardModel.Card.Sections["DocumentCommonInfo"].Fields["AuthorID"];
                cardModel.Card.Sections["PnrErrands"].Fields["ControllerName"] = cardModel.Card.Sections["DocumentCommonInfo"].Fields["AuthorName"];
            }

            return base.Initialized(context);
        }
    }
}

File: WebClient SDK/src/PnrScripts/UI/PnrErrandUIExtension.ts
import { Guid } from 'tessa/platform';
import { CardUIExtension, ICardUIExtensionContext } from 'tessa/ui/cards';
import { PnrCardTypes } from '../Shared/PnrCardTypes';

export class PnrErrandUIExtension extends CardUIExtension {
  public initialized(context: ICardUIExtensionContext) {
    if (!Guid.equals(context.card.typeId, PnrCardTypes.PnrErrandTypeID)) {
      return;
    }
    // const errandsControllers = context.card.sections.tryGet("PnrErrandsControllers");
    // TODO добавить добавление автора карточки в коллекционную секцию PnrErrandsControllers
  }
}